tests/simple-tests/Tests/Module/Data/CapNProto/Untyped.hs

```diff
@@ -24,14 +24,7 @@ untypedTests :: Test
 untypedTests = assertionsToTest "Untyped Tests"  $ map tst
     [ ( aircraftSchema
       , "Aircraft"
-      , [here|(f16 = (base = (
-            name = "bob",
-            homes = [],
-            rating = 7,
-            canFly = true,
-            capacity = 5173,
-            maxSpeed = 12.0,
-        )))|]
+      , "(f16 = (base = (name = \"bob\", homes = [none], rating = 7, canFly = true, capacity = 5173, maxSpeed = 12.0)))"
```

Comment:
What was the problem here exactly?

Reply:
The test on `capnp encode` fails with the error `"<stdin>:1: error: Parse error: Empty list item.\n<stdin>:1:1: error: Missing field name.\n"`. The main fix is changing `homes = []` to `homes = [none]` , so I'll add `here` back.

Reply:
Odd, it works on my machine (and travis & hydra). What version of capnp are you using?

Reply:
I'm using capnp 0.5.3. Upon closer inspection, it's the final comma after the `maxSpeed` field that it's choking on. Removing the comma gets rid of the error, I mistakenly thought it was due to the empty list. The problem only occurs when I use `cabal test` instead of `make nix-build`, so it could be due to differing versions.